vignettes/filling.Rmd

```diff
@@ -0,0 +1,676 @@
+)
+```
+
+:::: {.row .mt-3}
+
+::: {.col-md-6 .mt-auto .mb-auto}
+::: {.callout .callout-note}
+<h3 data-toc-skip>Resizable example</h3>
+
+Notice the resizing handle on the lower-right hand corner of the example above. Use it to change the size of the "window" and see the behavior of the filling plots  
+:::
+:::
+
+::: col-md-6
+::: {.callout .callout-note}
+<h3 data-toc-skip>Limiting shrinkage</h3>
+
+If you're worried about plots becoming too small, consider putting them in a `card_body()` with a `min_height` (like we do later on). Also, if you don't want the card border, you can do `card(class = "border-0", ...)`
+:::
+:::
+::::
+
+### Multiple columns
+
+Since `layout_column_wrap()` is a [fill item]{.b-blue} (by default), it grows/shrinks just like any other [fill item]{.b-blue}. It also defaults to `fillable = TRUE`, which in this case, means each column gets wrapped in a [fillable container]{.b-pink}. That's why, in this example, `plots[[1]]` and `plots[[1]]` also grow/shrink to match the size of the `layout_column_wrap()` container.
+
+```{r as_iframe=TRUE, resizable=TRUE}
+page_fillable(
+  h2("Diamond plots"),
+  layout_column_wrap(
+    width = 1/2,
+    plots[[1]], plots[[2]]
+  ),
+  plots[[3]]
+)
+```
+
+### Value boxes
+
+Since `value_box()` is a [fill item]{.b-blue} (by default), it grows/shrinks just like any other [fill item]{.b-blue}. This is especially useful for keeping a common baseline in a multi-column layout. That said, the multi-layout column that holds value boxes probably doesn't want it default `fill = TRUE` behavior, since the value boxes should be given more/less space and the window becomes larger/smaller:
+
+```{r as_iframe=TRUE, resizable=TRUE}
+boxes <- layout_column_wrap(
+  width = 1/3, fill = FALSE,
+  value_box(
+    "Total diamonds",
+    scales::comma(nrow(diamonds)),
+    showcase = bsicons::bs_icon("gem", size = NULL)
+  ),
+  value_box(
+    "Average price", 
+    scales::dollar(mean(diamonds$price), accuracy = 1),
+    showcase = bsicons::bs_icon("coin", size = NULL),
+    theme_color = "success"
+  ),
+  value_box(
+    "Average carat",
+    scales::number(mean(diamonds$carat), accuracy = .1),
+    showcase = bsicons::bs_icon("search", size = NULL),
+    theme_color = "dark"
+  )
+)
+
+page_fillable(
+  boxes,
+  layout_column_wrap(
+    width = 1/2,
+    plots[[1]], plots[[2]]
+  ),
+  plots[[3]]
+)
+```
+
+::: {.callout .callout-note}
+<h3 data-toc-skip>Column wrapping layouts</h3>
+
+To learn more about `layout_column_wrap()`, see [this article](column-layout.html).
+:::
+
+
+### Full-screen cards
+
+As eluded to in the [Carrying fill](#carrying-fill) section, `card()` and `card_body()` are [fill carriers]{.b-purple} (that is, they are both [fillable]{.b-pink} and [fill]{.b-blue}, by default). Therefore, by wrapping each plot in a card, the card not only grows/shrinks (since they are [fill]{.b-blue}), but also retain the plot's ability to grow/shrink (since they are [fillable]{.b-pink}).
```

Comment:
```suggestion
As alluded to in the [Carrying fill](#carrying-fill) section, `card()` and `card_body()` are [fill carriers]{.b-purple} (that is, they are both [fillable]{.b-pink} and [fill]{.b-blue}, by default). Therefore, by wrapping each plot in a card, the card not only grows/shrinks (since they are [fill]{.b-blue}), but also retain the plot's ability to grow/shrink (since they are [fillable]{.b-pink}).
```